samples/petclinic/README.md

```diff
@@ -0,0 +1,105 @@
+# Spring PetClinic with MySQL
+
+[Spring PetClinic][petclinic] is a sample [Spring Boot][boot] web application that can be used with MySQL.
+
+## Setup
+
+If not already installed, [install the ServiceBinding CRD and controller][install].
+
+## Deploy
+
+Apply the PetClinic application, MySQL service and connect them with a ServiceBinding:
+
+```sh
+kubectl apply -f ./samples/petclinic
+```
+
+Wait for the application (and database) to start and become healthy:
+
+```sh
+kubectl wait deployment petclinic --for condition=available --timeout=2m
+```
+
+## Understand
+
+Inspect the PetClinic application as bound:
+
+```sh
+kubectl describe deployment petclinic
+```
+
+If the ServiceBinding is working, a new environment variable (SERVICE_BINDINGS_ROOT), volume and volume mount (petclinic-db-projection-binding) is added to the deployment.
+The describe output will contain:
+
+```txt
+...
+  Containers:
+   application:
+    Image:      scothis/petclinic:service-bindings
+    Port:       <none>
+    Host Port:  <none>
+    Environment:
+      SPRING_PROFILES_ACTIVE:  mysql
+      SERVICE_BINDINGS_ROOT:   /bindings
+    Mounts:
+      /bindings/petclinic-db from petclinic-db-projection-binding (ro)
+  Volumes:
+   petclinic-db-projection-binding:
+    Type:        Secret (a volume populated by a Secret)
+    SecretName:  petclinic-db-projection
```

Comment:
```suggestion
    SecretName:  petclinic-db
```
`petclinic-db` is the name the sample config uses, no?

Reply:
`petclinic-db` is the name or the original secret, but it's content is copied into a new secret to facilitate mappings and setting the `type` and `provider`. The projected secret is named `petclinic-db-projection`, but the name may be generated in the future.